settings.gradle.kts

```diff
@@ -3,3 +3,4 @@ rootProject.name = "avito-android-infra"
 includeBuild("libraries")
 includeBuild("build-logic")
 includeBuild("subprojects")
+includeBuild("samples")
```

Comment:
In the future, it might be reasonable to put in a test runner project, as a part of it.

Reply:
yep, but also as composite build